pkg/release/minor.go

```diff
@@ -98,7 +88,7 @@ func mustFindMinor(version string, releases []v1alpha1.Release) v1alpha1.Release
 		if vv.Major != rv.Major {
 			continue
 		}
-		if vv.Minor-1 != rv.Minor {
+		if vv.Minor <= rv.Minor {
```

Comment:
This makes sure we can find the previous latest minor version with space in between.